ControlWork/Queue.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace ControlWork;
+
+public class ThreadSafetyQueue<T, P>
+    where P : IComparable
+{
+    private readonly List<QueueElement<T, P>> _elements = new();
+    private readonly object _locker = new();
+    private int _size = 0;
+
+    public void Enqueue(T element, P priority)
```

Comment:
```suggestion
        /// <summary>
        /// Добавляет значение в очередь
        /// </summary>
    public void Enqueue(T element, P priority)
```
По-хорошему, каждый публичный метод, свойство и т.д. должен быть задокументирован.

Reply:
в прошлых семестрах от нас это не требовали, так что я никак не мог знать:(

Reply:
Просто комментарий на будущее